opennms-base-assembly/src/main/filtered/bin/opennms

```diff
@@ -173,14 +174,28 @@ isRunning() {
 }
 
 getPid(){
-	__pid=""
+	local __pid
+
 	if [ -f "$OPENNMS_PIDFILE" ]; then
 		__pid="$(cat "$OPENNMS_PIDFILE")"
 	fi
 	if [[ "$__pid" =~ ^[0-9]+$ ]] && [ "$__pid" -gt 0 ]; then
 		echo "$__pid"
 		return
 	fi
+
+	echo -1
+}
+
+findPid() {
+	local __matched_pid;
+
+	__matched_pid="$(pgrep -f _OPENNMS_LAUNCH 2>/dev/null || :)"
```

Comment:
what is that final colon doing?

Reply:
"or do nothing" so that `set -e` doesn't short circuit

equivalent of doing `|| true` but it's an actual no-op rather than a "true" value